navigator.lua

```diff
@@ -24,16 +24,16 @@ local settings = {
   key_navforward = "RIGHT",
   key_navopen = "ENTER",
   key_navclose = "ESC",
+  key_navpageup = "PGUP",
+  key_navpagedown = "PGDWN",
 
   --fallback if no file is open, should be a string that points to a path in your system
   defaultpath = windows_desktop or os.getenv("HOME") or "/",
-  forcedefault = false, --force navigation to start from defaultpath instead of currently playing file
+  forcedefault = true, --force navigation to start from defaultpath instead of currently playing file
   --favorites in format { 'Path to directory, notice trailing /' }
   --on windows use double backslash c:\\my\\directory\\
   favorites = {
-    '/media/HDD2/music/music/',
-    '/media/HDD/users/anon/Downloads/',
-    '/home/anon/',
+    'y:\\Anime\\',
```

Comment:
Could you undo the changes to settings as they are not relevant for the PR

Reply:
Reverted back to original default configurations.